### ISSUE TITLE
valid-jsdoc incorrectly flags multi-level params as duplicates

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#661]
In JSDoc, you can do this:

``` js
/**
 * @param {Object} options description
 * @param {string} options.name description
 */
```

Right now, `valid-jsdoc` will warn that the param `options` is defined twice. This is a valid pattern and should not be flagged.
